src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/ingress/of/IngressFlowModFactory.java

```diff
@@ -213,11 +234,42 @@ public OFFlowMod makeDefaultPortServer42IngressFlowMessage(int server42UpdPortOf
 
         OFFlowMod.Builder builder = flowModBuilderFactory.makeBuilder(of, TableId.of(SwitchManager.INGRESS_TABLE_ID),
                 SERVER_42_INGRESS_DEFAULT_FLOW_PRIORITY_OFFSET)
-                .setCookie(U64.of(buildServer42IngressCookie().getValue()))
                 .setMatch(match);
         return makeServer42IngressFlowMessage(builder, Collections.emptyList());
     }
 
+    /**
+     * Make rule to match server 42 packets by inner VLAN tag and forward in in ISL.
+     */
+    public OFFlowMod makeDoubleServer42IngressFlowMessage() {
+        FlowEndpoint endpoint = command.getEndpoint();
+        RoutingMetadata metadata = buildServer42IngressMetadata();
+        OFFlowMod.Builder builder = flowModBuilderFactory
+                .makeBuilder(of, TableId.of(SwitchManager.INGRESS_TABLE_ID),
+                        SERVER_42_INGRESS_DOUBLE_VLAN_FLOW_PRIORITY_OFFSET)
```

Comment:
This constant ruin all efforts of making priorities clear (at least more clear than original design). The idea was to define some base priority and let code to define positive an negarive offsets for this base. As the result - clear vision of order OF flows will be used.

Reply:
But this constant is offset. And makeBuilder will calculate priority by `(basePriority + priorityOffset)`

Reply:
Yes and it hides offset from view. You must find it's definition and see / remember value to compare with other rules. My opinion - it must be raw number.

To be more clear we should have somethig like `.makeBuilder(of, tableId, PRIOTITY_STEP * N)` where N is desired step and PRIORITY_STEP is 10 (allows to add in futue rules between existing rules).